pkg-manifest/exportable-manifest/src/index.ts

```diff
@@ -178,3 +178,31 @@ async function replaceWorkspaceProtocolPeerDependency (depName: string, depSpec:
 
   return depSpec.replace('workspace:', '')
 }
+
+async function replaceJsrProtocol (depName: string, depSpec: string): Promise<string> {
+  if (!depSpec.startsWith('jsr:')) {
+    return depSpec
+  }
+
+  depSpec = depSpec.slice('jsr:'.length)
+
+  // syntax: jsr:<tag>
```

Comment:
Your use of "tag" is incorrect. It is a "spec" (short for "version specifier). A spec may be one of three: version, range, tag.